cmd/zpool/zpool.d/smart

```diff
@@ -45,7 +50,7 @@ if [ -b "$VDEV_UPATH" ] && [ -x "$smartctl_path" ]; then
 	# SATA:
 	#	ATA Version is:   8
 	#
-	type=$(echo "$raw_out" | grep -m 1 -Eo '^ATA|SAS$')
+	type=$(echo "$raw_out" | grep -m 1 -Eo '^ATA|SAS')
```

Comment:
I've been thinking about this.  If we set type in the awk section, do we need to here?  We might want to revise the -z type section below as well.

Reply:
Yea, we could grep `type`  from `$out` so that we're only setting it in the awk section.